<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>428</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>744</width>
    <height>428</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ДКВС</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Размах интерферограммы</string>
   </property>
  </widget>
  <widget class="QLabel" name="lYRange">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Длина интерферограммы</string>
   </property>
  </widget>
  <widget class="QLabel" name="lXRange">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lTemp">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>9</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Температура</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPoints">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCustomPlot" name="cpGraph" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>79</y>
     <width>721</width>
     <height>311</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelPause">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Пауза</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="cpGraph_2" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox1">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,0,0">
    <item>
     <widget class="QPushButton" name="pushButtonPause">
      <property name="text">
       <string>Пауза</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSave">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>lYRange</zorder>
  <zorder>label_3</zorder>
  <zorder>lXRange</zorder>
  <zorder>lTemp</zorder>
  <zorder>label_4</zorder>
  <zorder>lPoints</zorder>
  <zorder>cpGraph</zorder>
  <zorder>buttonBox1</zorder>
  <zorder>label</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox1</sender>
   <signal>rejected()</signal>
   <receiver>Dialog2</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
